ftplugin/ps1xml.vim

```diff
@@ -14,6 +14,11 @@ let b:did_ftplugin = 1
 setlocal tw=0
 setlocal commentstring=#%s
 setlocal formatoptions=tcqro
+" Enable autocompletion of hyphenated PowerShell commands,
+" e.g. Get-Content or Get-ADUser
+setlocal iskeyword+=-
```

Comment:
Just push a new commit. It's just a pain if you abandon this and open a new one. All information about this is lost. So fix it, commit, and push all in the same branch from which you created this PR in the first place.